homeassistant/components/sonos/__init__.py

```diff
@@ -225,6 +230,24 @@ def _async_add_visible_zones(subscription_succeeded: bool = False) -> None:
 
         async def async_subscription_failed(now: datetime.datetime) -> None:
             """Fallback logic if the subscription callback never arrives."""
+            addr, port = sub.event_listener.address
+            listener_address = f"{addr}:{port}"
+            if advertise_ip := soco_config.EVENT_ADVERTISE_IP:
+                listener_address += f" (advertising as {advertise_ip})"
+            ir.async_create_issue(
```

Comment:
When does it clean it up?

Reply:
It would require a restart or reload of the integration. Will that not clear the issue?

Reply:
A restart will, but a reload won't.